sources/core/objects/turfs/MetalWall.cpp

```diff
@@ -45,7 +111,27 @@ ReinforcedWall::ReinforcedWall()
     SetVisibleLevel(visible_level::HIGH_TURF);
 
     SetSprite("icons/walls.dmi");
-    SetState("r_wall");
 
     SetName("Reinforced wall");
+
+    default_state_="rwall";
+    current_state_=0;
+
+    SetState("rwall0");
+}
+
+void ReinforcedWall::AfterWorldCreation()
```

Comment:
This one should be removed too (because `MetalWall` one does the same thing).
Also, `AfterWorldCreation` should call firstly base class `AfterWorldCreation`.